typescript/aleo-sdk/src/clients/base.ts

```diff
@@ -40,12 +45,14 @@ export class AleoBase {
     );
     assert(rpcUrls.length > 0, `got no rpcUrls`);
 
-    this.rpcUrls = rpcUrls;
+    this.rpcUrls = rpcUrls.map((r) =>
+      r.replaceAll('/testnet', '').replaceAll('/mainnet', ''),
+    );
```

Comment:
A comment explaining why this is needed would help here

Reply:
agree